From ce68fc7826bf190abaa77decbf8093a3f020bd3b Mon Sep 17 00:00:00 2001
From: Leonidas Spyropoulos <artafinde@archlinux.org>
Date: Fri, 7 Apr 2023 14:56:57 +0300
Subject: [PATCH] feat: move version in gradle.properties

This makes it easier to set the version from the command line when we
build.

Signed-off-by: Leonidas Spyropoulos <artafinde@archlinux.org>
---
 build.gradle      | 1 -
 gradle.properties | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/build.gradle b/build.gradle
index a2ea3d0..31b2fbb 100644
--- a/build.gradle
+++ b/build.gradle
@@ -13,7 +13,6 @@ configurations {
 }
 
 group 'org.jboss.aerogear'
-version '3.0.1-SNAPSHOT'
 
 apply plugin: 'java'
 
diff --git a/gradle.properties b/gradle.properties
index 2fb4e20..bf0ce7f 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -1,2 +1,3 @@
 keycloakVersion=21.0.1
 prometheusVersion=0.16.0
+version=3.0.1-SNAPSHOT
